src/Microsoft.Health.Fhir.CosmosDb/Features/Search/Queries/QueryBuilderOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Core.Features.Search.Expressions;
+
+namespace Microsoft.Health.Fhir.CosmosDb.Features.Search.Queries
+{
+    internal class QueryBuilderOptions
+    {
+        public QueryBuilderOptions(IReadOnlyList<IncludeExpression> includes = null, QueryProjection projection = QueryProjection.Default)
+        {
+            Includes = includes;
```

Comment:
>includes [](start = 23, length = 8)

nit: Includes = includes?? Array.Empty<IncludeExpression>()
and you don't have to write that part of code in service.
A bit hacky, but I think it would make code more cleaner.